examples/jsm/loaders/NRRDLoader.js

```diff
@@ -445,34 +443,25 @@ class NRRDLoader extends Loader {
 
 		}
 
-
 		if ( ! headerObject.vectors ) {
 
-			volume.matrix.set(
-				1, 0, 0, 0,
-				0, 1, 0, 0,
-				0, 0, 1, 0,
-				0, 0, 0, 1 );
+			volume.matrix = volume.inverseMatrix = new Matrix4();
```

Comment:
This doesn't seem right?

Reply:
Both properties point to the same object now.

Reply:
Why do they have to point to the same matrix?

Reply:
Just because it looks a little bit simpler in one line. But OK, I can make it two lines.

Reply:
@LeviPesin Do you mind confirming that you understand what @Mugen87 wrote above? 😬

Reply:
I guess the thing to understand is that these kind of optimizations can easily create very confusing bugs.

Reply:
OK. It is changed now.